packages/core/src/app/address/AddressSelect.tsx

```diff
@@ -1,65 +1,25 @@
 import { Address, CustomerAddress } from '@bigcommerce/checkout-sdk';
-import React, { FunctionComponent, memo, PureComponent, ReactNode } from 'react';
+import React, { FunctionComponent, memo } from 'react';
 
 import { TranslatedString } from '@bigcommerce/checkout/locale';
 
 import { preventDefault } from '../common/dom';
 import { DropdownTrigger } from '../ui/dropdown';
 
-import isEqualAddress from './isEqualAddress';
-import './AddressSelect.scss';
 import AddressSelectButton from './AddressSelectButton';
+import isEqualAddress from './isEqualAddress';
+import { PoweredByPaypalConnectLabel, usePayPalConnectAddress } from './PayPalAxo';
```

Comment:
This PR looks good to me.
Does it make sense to create a PalPalAxo package?
cc @animesh1987

Reply:
This changes will be moved in Braintree package in future.

Reply:
Maybe we can create `Braintree` package now so that we don't forget to come back here.

Reply:
Hey @bc-peng. We created a task for that and I am planing to move this part to PayPalConnect/PayPalAxo package in i19 sprint.

Is it ok for you?

Reply:
We will clean up core package from paypal specific code as soon as possible